src/nixos-anywhere.sh

```diff
@@ -619,7 +625,9 @@ if [[ ${phases[reboot]} == 1 ]]; then
   if command -v zpool >/dev/null && [ "\$(zpool list)" != "no pools available" ]; then
     # we always want to export the zfs pools so people can boot from it without force import
     umount -Rv /mnt/
-    swapoff -a
+    if [[ ${swapoff} == "y" ]]; then
+      swapoff -a
```

Comment:
What I don't like about this change is that it breaks the export the when people have swap on ZFS.
People shouldn't need to have adding a flag to make nixos-anywhere work.

Reply:
I think something like this is more sensible:

```suggestion
    swapon --show --noheadings | awk '/^\/dev\/zd/ {print $1}' | while read device; do
      swapoff "$device"
    done
```

You would need to have your system not have swap in zfs. However you need to do this anyway because with 1GB you would run into deadlock issues with zfs if you would try to have it in the pool.